### ISSUE TITLE
FRONTEND: Give an option in equation explorer to allow the "expanded" list of equations beyond the original 4694

### ISSUE THREAD

**Issue by teorth** [teorth/equational_theories#378]
See https://leanprover.zulipchat.com/#narrow/stream/458659-Equational/topic/Equation.205105.20-.3E.20Equation.202 for discussion.  By default one should restrict attention to the "Core" graph of the initial 4694 equations.

**Comment by teorth** [comment]
This may require first resolving #142.

**Comment by vlad902** [comment]
Some of our workflows depend on the sporadic equations not being included in the list of 'not fully solved equations'. It would probably require yet-another checkbox to filter them out to maintain those workflows, unless there's another method I'm not thinking of.

**Comment by teorth** [comment]
Yeah, it requires yet another checkbox, and they should filtered out by default.  We're beginning to have quite a few of these checkboxes, but they're all independently useful so I don't see any better thing to do than just have all of the options available.

**Comment by Osalotioman** [comment]
claim

**Comment by Osalotioman** [comment]
disclaim

**Comment by Chessing234** [comment]
propose #1489
